browser/brave_wallet/brave_wallet_event_emitter_browsertest.cc

```diff
@@ -142,3 +182,33 @@ IN_PROC_BROWSER_TEST_F(BraveWalletEventEmitterTest,
              content::EXECUTE_SCRIPT_USE_MANUAL_REPLY);
   EXPECT_EQ(base::Value(true), result_first.value);
 }
+
+IN_PROC_BROWSER_TEST_F(BraveWalletEventEmitterTest,
+                       CheckForAnAccountChangedEvent) {
+  RestoreWallet();
+  GURL url =
+      https_server()->GetURL("a.com", "/brave_wallet_event_emitter.html");
+  ASSERT_TRUE(ui_test_utils::NavigateToURL(browser(), url));
+  content::WebContents* contents =
+      browser()->tab_strip_model()->GetActiveWebContents();
+  WaitForLoadStop(contents);
```

Comment:
This is redundant because `NavigateToURL` is blocking until the navigation finishes.

Reply:
ok, I will try that

Reply:
done!